metricbeat/module/iis/webserver/manifest.yml

```diff
@@ -0,0 +1,123 @@
+default: true
+input:
+  module: windows
+  metricset: perfmon
+  defaults:
+    perfmon.group_measurements_by_instance: true
+    perfmon.ignore_non_existent_counters: true
+    perfmon.group_all_counter: "worker_process_count"
+    perfmon.counters:
+    #network
+    - instance_label: ''
+      measurement_label: network.total_bytes_received
+      query: '\Web Service(_Total)\Total Bytes Received'
+    - instance_label: ''
+      measurement_label: network.total_bytes_sent
+      query: '\Web Service(_Total)\Total Bytes Sent'
+    - instance_label: ''
+      measurement_label: network.bytes_sent_per_sec
+      query: '\Web Service(_Total)\Bytes Sent/sec'
+    - instance_label: ''
+      measurement_label: network.bytes_received_per_sec
+      query: '\Web Service(_Total)\Bytes Received/sec'
+    - instance_label: ''
+      measurement_label: network.current_connections
+      query: 'Web Service(_Total)\Current Connections'
+    - instance_label: ''
+      measurement_label: network.maximum_connections
+      query: '\Web Service(_Total)\Maximum Connections'
+    - instance_label: ''
+      measurement_label: network.total_connection_attempts
+      query: '\Web Service(_Total)\Total Connection Attempts (all instances)'
+    - instance_label: ''
+      measurement_label: network.total_get_requests
+      query: '\Web Service(_Total)\Total Get Requests'
+    - instance_label: ''
+      measurement_label: network.get_requests_per_sec
+      query: '\Web Service(_Total)\Get Requests/sec'
+    - instance_label: ''
+      measurement_label: network.total_post_requests
+      query: '\Web Service(_Total)\Total Post Requests'
+    - instance_label: ''
+      measurement_label: network.post_requests_per_sec
+      query: '\Web Service(_Total)\Post Requests/sec'
+    - instance_label: ''
+      measurement_label: network.total_delete_requests
+      query: '\Web Service(_Total)\Total Delete Requests'
+    - instance_label: ''
+      measurement_label: network.delete_requests_per_sec
+      query: '\Web Service(_Total)\Delete Requests/sec'
+    - instance_label: ''
+      measurement_label: network.service_uptime
+      query: '\Web Service(_Total)\Service Uptime'
```

Comment:
We can also add error counters:
`Web Service(*)Total Not Found Errors`
`Web Service(*)Not Found Errors/sec`
`Web Service(*)Total Locked Errors`
`Web Service(*)Locked Errors/sec`